lib/validators/movies/list.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().lowercase().optional(),
+  release_year: Joi.number().optional()
```

Comment:
When the `release_year` is added to the filter, it will add something like `WHERE release_year = 2006` to the SQL query that fetched the movies. Because the `release_year` column is an integer in the database, something like `WHERE release_year = 2006.2` will create an error because Postgres is unable to coerce 2006.2 into an integer. You can see that this bubbles up into a 500 error if you change your movies integration list test to hit the url `/movies?release_year=2006.2`.

I think the appropriate remedy in this case is to ensure here that `release_year` is an integer:

```
release_year: Joi.number().integer().optional()
```

It looks like this is also a problem in your create endpoint, so you should also add `.integer()` to the create validator.